Dockerfile.ui

```diff
@@ -35,7 +36,7 @@ COPY cvat-core/ /tmp/cvat-core/
 COPY cvat-canvas3d/ /tmp/cvat-canvas3d/
 COPY cvat-canvas/ /tmp/cvat-canvas/
 COPY cvat-ui/ /tmp/cvat-ui/
-RUN CLIENT_PLUGINS="${CLIENT_PLUGINS}" UI_APP_CONFIG="${UI_APP_CONFIG}" yarn run build:cvat-ui
+RUN CLIENT_PLUGINS="${CLIENT_PLUGINS}" UI_APP_CONFIG="${UI_APP_CONFIG}" REACT_APP_CVAT_BILLING_URL="${REACT_APP_CVAT_BILLING_URL}" yarn run build:cvat-ui
```

Comment:
all environment variables which name start with ``REACT_APP_`` are added to build automatically 
Plus core app does not need to know about this var. Could you please explain why it was added?

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables/